sdk/storage/azblob/blob/client.go

```diff
@@ -362,14 +362,14 @@ func (b *Client) DownloadStream(ctx context.Context, o *DownloadStreamOptions) (
 		count = *o.Count
 	}
 
-	eTag := ""
+	eTag := azcore.ETag("")
```

Comment:
This looks dubious.  Shouldn't we default to a `nil` ETag?

Reply:
Great catch. Yes make it nil. My guess is that it would work anyway but nil is technically correct.

Reply:
Fixed